go/database/log/migrations/20240401221053_initial.sql

```diff
@@ -0,0 +1,15 @@
+-- Create "collection" table
+CREATE TABLE "public"."collection" (
+  "id" text NOT NULL,
+  "record_compaction_offset_position" bigint NOT NULL,
+  "record_enumeration_offset_position" bigint NOT NULL,
+  PRIMARY KEY ("id")
+);
+-- Create "record_log" table
+CREATE TABLE "public"."record_log" (
+  "offset" bigint NOT NULL,
+  "collection_id" text NOT NULL,
+  "timestamp" integer NOT NULL DEFAULT (EXTRACT(epoch FROM now()))::integer,
```

Comment:
What is the meaning of this timestamp? Can we name it so that the meaning is more explict?

Reply:
It was there before, tbh I don't know how is it use down stream.

Reply:
Base on your explanation @Ishiihara, it is really the timestamp of insertion. Do you have another name in mind?